llama-index-integrations/vector_stores/llama-index-vector-stores-opensearch/llama_index/vector_stores/opensearch/base.py

```diff
@@ -69,6 +70,7 @@ def __init__(
         text_field: str = "content",
         method: Optional[dict] = None,
         engine: Optional[str] = "nmslib",
+        space_type: Optional[str] = "l2",
```

Comment:
What does `space_type` mean? What does `12` mean? (is this just the naming/value that open search uses? Just confusing for someone out of the loop haha)

Reply:
space type is basically the distance metric that opensearch uses between vectors. they use the term space type for that. l2 is one of the distance functions. more detail here https://opensearch.org/docs/latest/search-plugins/knn/knn-score-script/

Reply:
https://opensearch.org/docs/latest/search-plugins/knn/knn-score-script/#spaces better link lol

Reply:
ohhhh its L2, not 12 lol

Reply:
that makes way more sense